tests/test_distro.py

```diff
@@ -33,11 +33,11 @@
 
 IS_LINUX = sys.platform.startswith("linux")
 if IS_LINUX:
-    import distro
+    import distro.distro
```

Comment:
How about `from distro import distro`?

Reply:
That works for me and results in a smaller diff. I made this change in the latest revision.